### PR TITLE
Modified functionality to be correct

### PR DESCRIPTION
**Pull request by XOmniverse**
These policies incorrectly reported on the number of instances of each tag value, which is not cardinality. This updates them to properly report the number of unique values for each key instead.

Both policies have been tested.

### CODE REVIEW

File: operational/aws/tag_cardinality/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.1
+
+- policy now reports cardinality proper instead of tag values directly
```
Comment (nia-vf1) [inline operational/aws/tag_cardinality/CHANGELOG.md:5]: Did you mean to say 'properly' rather than 'proper'? Same for the other changelog file

**Review by bryankaraffa — COMMENTED** [review]
Combing backlog of reviews requested, is this is something you are trying to merge still ?

File: operational/aws/tag_cardinality/CHANGELOG.md (hunk 1 above)
Comment (XOmniverse) [reply operational/aws/tag_cardinality/CHANGELOG.md:5]: No. See definition 4 here: https://www.collinsdictionary.com/us/dictionary/english/proper

**Review by nia-vf1 — APPROVED** [review]
LGTM

**Pull request merged**